src/components/ui/formik/TextAreaField.tsx

```diff
@@ -4,11 +4,19 @@ import { FormGroup, TextArea } from '@patternfly/react-core';
 import { TextAreaProps } from './types';
 import { getFieldId } from './utils';
 
-const TextAreaField: React.FC<TextAreaProps> = ({ label, helperText, isRequired, ...props }) => {
+const TextAreaField: React.FC<TextAreaProps> = ({
+  label,
+  helperText,
+  isRequired,
+  children,
+  ...props
+}) => {
   const [field, { touched, error }] = useField(props.name);
   const fieldId = getFieldId(props.name, 'input');
   const isValid = !(touched && error);
   const errorMessage = !isValid ? error : '';
+  const handleAreaChange = React.useCallback((value, event) => field.onChange(event), [field]);
```

Comment:
I think using useCallback here is not helping the performance
https://kentcdodds.com/blog/usememo-and-usecallback